src/core/controllers/controllers_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/goharbor/harbor/src/common/models"
 	utilstest "github.com/goharbor/harbor/src/common/utils/test"
 	"github.com/goharbor/harbor/src/core/config"
-	"github.com/goharbor/harbor/src/core/proxy"
+	"github.com/goharbor/review/harbor/src/core/proxy"
```

Comment:
This import path may not exist. Should it be `github.com/goharbor/src/core/proxy` ?

Reply:
ok thanks.